game/shared/vgen.cc

```diff
@@ -14,20 +14,20 @@
 
 constexpr static std::int64_t FLOATLANDS_VAR = INT64_C(64);
 constexpr static std::int64_t FLOATLANDS_MIDDLE = INT64_C(640);
-constexpr static std::int32_t FLOATLANDS_START_C = INT32_C(2); // 16
+constexpr static std::int32_t FLOATLANDS_START_C = INT32_C(2); // 32 blocks
```

Comment:
These are commented out previous values, not magic constants

Reply:
Also it's a bit on me for not using `ChunkCoord::value_type` and `VoxelCoord::value_type`